src/tests/algorithms_test/pid_MergeParticleID.cc

```diff
@@ -125,6 +125,9 @@ TEST_CASE("the PID MergeParticleID algorithm runs", "[MergeParticleID]") {
         return obj;
     }
     FAIL("ERROR: cannot find CherenkovParticleID given track");
+    if(coll->empty())
```

Comment:
Surprisingly, despite PODIO declaring that they try to mimic STL collections, there is no `empty()`: https://github.com/AIDASoft/podio/blob/master/python/templates/Collection.h.jinja2

Let's revert
```suggestion
    if(coll->size() == 0)
```

Reply:
Ah shoot, yes, it's not even the first time I've suggested this in a code review and we've had to revert.

Reply:
https://github.com/AIDASoft/podio/issues/476